src/Microsoft.VisualBasic.Core/tests/VB/Microsoft.VisualBasic.VB.Tests.vbproj

```diff
@@ -11,4 +11,8 @@
     <Compile Include="FileIOTests.vb" />
     <Compile Include="TextFieldParserTests.vb" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsUap)' == 'true'">
+    <!-- UAP is missing default imports -->
+    <Import Include="Microsoft.VisualBasic;System;System.Collections;System.Collections.Generic;System.Diagnostics;System.Linq;System.Xml.Linq;System.Threading.Tasks" />
```

Comment:
I was a bit surprised I needed to do this.  The reason was that the SDK conditioned these :(
https://github.com/dotnet/sdk/blob/081102d8176c2788928a416e6d4dd646aa8e723e/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.targets#L20-L43

Reply:
Should we log an issue against htem?

Reply:
I don't know if the SDK technically supports targeting UAP.  If it were up to me I'd recommend that instead we change the source to not rely on global imports.

Reply:
@KathleenDollard might know or care about this behavior.